src/renderer/components/Wallet/WalletBusiness/WalletBusinessBridgechains.vue

```diff
@@ -122,12 +127,18 @@ export default {
      * is received
      */
     async loadBridgechains () {
-      if (!this.wallet_fromRoute || this.isFetching) {
-        return
-      }
+      try {
+        if (!this.wallet_fromRoute || this.isFetching) {
+          return
+        }
 
-      this.isLoading = true
-      this.fetchBridgechains()
+        this.isLoading = true
+        this.fetchBridgechains()
+      } catch (error) {
+        throw new Error('It is not possible load bridgechain list')
```

Comment:
I would log instead of throwing an error. I've experienced it where throwing an error has halted components meaning they don't work properly

Reply:
make sense :+1: I'll add the log